.github/workflows/first-pr.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 permissions:
   contents: read
-  issues: read
+  issues: write
```

Comment:
To follow the security guidelines, the `write` permissions should not be at job level instead?

Reply:
As discussed this workflow contain and will contain only one job so it's equivalent